client/src/components/CreateAccount.js

```diff
@@ -1,42 +1,95 @@
 import React,{ useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import API from '../API'
 
 export const CreateAccount = () => {
     useEffect(() => {
         document.body.className = 'bodyLogin';
         return () => { document.body.className = ''; }
-      });
-
-  return (
-    <div className = 'containerLogin'>
-    <div className = 'CreateAccount'>
-        <div class='createaccTitle'>Create account</div>
-        <form className = 'add-form'>
-            <div className= 'form-control'>
-                <label>Your Name*</label>
-                <input required type='text' placeholder = 'Enter Full Name'/>
-            </div>
-            <div className= 'form-control'>
-                <label>UCSD Email*</label>
-                <input required type='text' placeholder = 'Enter Email' />
-            </div>
-            <div className= 'form-control'>
-                <label>Password*</label>
-                <input required type='text' placeholder = 'Password' />
-            </div>
-            <div className= 'form-control'>
-                <label>Re-enter Password*</label>
-                <input required type='text' placeholder = 'Re-type password' />
-            </div>
-        </form>
-        <button type="button" className = 'createbtn'>Create your account</button>
-        <div className = 'backtoLogin'>
-            <div style={{marginBottom: "10px"}}>* Required Fields</div>
-            <div>Already have an account?</div> 
-            <a href='/'> Back to Login</a>
-        </div> 
-    </div>
-    </div>
-  )
+    });
+
+    const navigate = useNavigate();
+    
+    const createUser = async (e) => {
+        e.preventDefault();
+        const payload = {
+            user: {
+                firstName: e.target.firstname.value,
+                lastName: e.target.lastname.value,
+                email: e.target.email.value,
+                password: e.target.password.value,
+            }
+        }
+       const response = await API.createUser(payload);
+       if(response.status === 200){
+           alert('Created successfully');
+           navigate('/');
+       }
```

Comment:
I don't think we need to alert a user when their account has been created (I haven't really seen something like this in a standard login flow). I think just navigating should be fine